packages/pulpcore/python-django/python-django.spec

```diff
@@ -46,14 +46,20 @@ popd
 %install
 %py3_install
 
+# rename django-admin so we don't conflict with python2-django
+mv %{buildroot}%{_bindir}/django-admin %{buildroot}%{_bindir}/python3-django-admin
```

Comment:
the name was chosen with a fair dice (and the fact that the Fedora packages actually ship that as a "compat" symlink to `django-admin`)